components/atom/tag/src/styles/index.scss

```diff
@@ -219,6 +223,18 @@ $base-class: '.sui-AtomTag';
         fill: $bgc;
       }
 
+      & > #{$self}-closeable {
+        color: $c-icon-close;
+      }
+
+      #{$self}-closeable > #{$self}-closeableIcon {
```

Comment:
```suggestion
      #{$self}-actoin > #{$self}-actionIcon {
```

Reply:
same here the class is sui-AtomTag-closeableIcon

Reply:
https://github.com/SUI-Components/sui-components/blob/b22b29db13ed763aeb08f9d39ad24acf3bf0309c/components/atom/tag/src/Standard.js#L36